app/tests/test_queries.py

```diff
@@ -327,7 +327,212 @@ def test_browse_data_get_specific_page_results(self, client: FlaskClient):
+                files[8].file_consignments.consignment_series.SeriesId,
+                "test series9",
+                "15/02/2023",
+                1,
+                1,
+            ),
+        ]
+
+    def test_browse_with_date_to_filter(self, client: FlaskClient):
+        """
+        Given multiple File objects in the database and consignment transfer complete date
+            that matches all of them
+        When browse_data is called with date_to
+        Then it returns a list containing multiple dictionary for the matching record with
+            expected fields
+        """
+        files = create_multiple_test_records()
+
+        filters = {"date_range": {"date_to": "28/02/2023"}}
+        result = browse_data(page=1, per_page=per_page, filters=filters)
+
+        assert result.items == [
+            (
+                files[0].file_consignments.consignment_bodies.BodyId,
+                "test body1",
+                files[0].file_consignments.consignment_series.SeriesId,
+                "test series1",
+                "01/01/2023",
+                1,
+                1,
+            ),
+            (
+                files[1].file_consignments.consignment_bodies.BodyId,
+                "test body2",
+                files[1].file_consignments.consignment_series.SeriesId,
+                "test series2",
+                "01/01/2023",
+                1,
+                1,
+            ),
+            (
+                files[9].file_consignments.consignment_bodies.BodyId,
+                "testing body10",
+                files[9].file_consignments.consignment_series.SeriesId,
+                "test series10",
+                "01/01/2023",
+                1,
+                1,
+            ),
+            (
+                files[10].file_consignments.consignment_bodies.BodyId,
+                "testing body11",
+                files[10].file_consignments.consignment_series.SeriesId,
+                "test series11",
+                "01/01/2023",
+                1,
+                2,
+            ),
+            (
+                files[2].file_consignments.consignment_bodies.BodyId,
+                "testing body3",
+                files[2].file_consignments.consignment_series.SeriesId,
+                "test series3",
+                "01/01/2023",
+                1,
+                1,
+            ),
+        ]
+
+    def test_browse_with_date_from_and_to_filter(self, client: FlaskClient):
+        """
+        Given multiple File objects in the database and consignment transfer complete date
+            that matches all of them
+        When browse_data is called with date_from and date_to
+        Then it returns a list containing multiple dictionary for the matching record with
+            date in between the range
+        """
+        files = create_multiple_test_records()
+
+        filters = {
+            "date_range": {"date_from": "01/02/2023", "date_to": "28/02/2023"}
```

Comment:
it would be ideal to change the date range or the setup data so that the objects returned here make it clear that the from and to filters are working in conjunction. Currently this results in the same assertion as in `def test_browse_with_date_from_filter(self, client: FlaskClient):`

Reply:
changes will be made to the test in AYR-604 branch